JASP-Engine/JASP/R/anovarepeatedmeasures.R

```diff
@@ -917,12 +919,12 @@ AnovaRepeatedMeasures <- function(jaspResults, dataset = NULL, options) {
           levelBNoDots <- gsub(.unv(comparisons[[compIndex]][2]), pattern = "\\.", replacement = " ")
           facLevelNoDots <- gsub(longData[[var]], pattern = "\\.", replacement = " ")
 
-          x <- subset(longData, facLevelNoDots == levelANoDots)
+          x <- subset(longData, gsub("X", "", facLevelNoDots) == gsub("X", "", levelANoDots))
```

Comment:
so why is there a `gsub` here? Because integer factors are automatically converted to X1, X2, etc? Maybe add a comment so that in the future we still understand what's going on.

Reply:
Yes they get introduced by emmeans